### DISCUSSION TITLE
Destination ISP for ports??

### DISCUSSION THREAD

**Discussion by dandrus104** [mozilla/hubs#3474] [General Questions & Discussion] [answered]
Does anyone know the destination ISP for the ports instead of opening our firewall to all of AWS? As a follow up to this discussion https://github.com/mozilla/hubs/discussions/3288 about which ports need to be opened

**Comment by robinkwilson** [comment, marked as answer]
**Are you having trouble connecting to a Hubs Cloud instance or hubs.mozilla.com?** We have an open bug to change our use of port 80 so our users with restrictive firewalls can connect.

You can see what ports are used in the aws security groups on the Cloudformation template here: https://github.com/mozilla/hubs-ops/blob/82e6f5cbef0ced07f51576fc8f72ea7c968a46af/cloudformation/stack.yaml#L363

"External" means what clients connect to. 
"Internal" means it's used between boxes - so no need to open those ones - unless they match up.

Currently you need (as mentioned in the other thread): 
443
80 - will switch to 8443 in the very near future with an upcoming template update
5349
51610 - 60999

**Reply by robinkwilson** [reply]
Marking as answer to remove thread from our pipeline, feel free to "Unmark as answer".
